Issues installing package

I am very excited to have found this package.  However, I have tried three approaches to installing it and none of them succeed.

1) Download the zip file of the repo.   Unzip, npm install, npm test fails:  **istanbul cover ./node_modules/tape/bin/tape ./test/*.js  | node_modules/tap-spec/bin/cmd.js**

2) npm install /hapi-login-example-postgres --save.  Fails to install package.json file.

3) git clone git@github.com:dwyl/hapi-login-example-postgres.git.  Fails with: **Permission denied (publickey).
fatal: Could not read from remote repository.**

Thanks!

I installed Postgres and created a database named test so that should not be an issue.  I added it and the secret key to the config file.

The error from "npm test":

```
> hapi-login-example-postgres@1.0.14 test /home/bmiller/projects/hapi-login-example-postgres-master
> istanbul cover ./node_modules/tape/bin/tape ./test/*.js  | node_modules/tap-spec/bin/cmd.js

No coverage information was collected, exit without writing coverage information
module.js:341
    throw err;
    ^

Error: Cannot find module '../node_modules/hapi-postgres-connection/node_modules/pg/lib/index.js'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/home/bmiller/projects/hapi-login-example-postgres-master/test/_create_table.test.js:8:10)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Object.Module._extensions.(anonymous function) [as .js] (/home/bmiller/projects/hapi-login-example-postgres-master/node_modules/istanbul/lib/hook.js:109:37)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)


npm ERR! Test failed.  See above for more details.

```

@nelsonic - I cloned the repo, ran `npm install`, and then `npm run create`.  The following is the error from `npm run create`.

```
[bmiller@jazz hapi-login-example-postgres]$ npm run create

> hapi-login-example-postgres@1.0.14 create /home/bmiller/projects/hapi-login-example-postgres
> node test/_create_table.js

module.js:341
    throw err;
    ^

Error: Cannot find module '../node_modules/hapi-postgres-connection/node_modules/pg/lib/index.js'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/home/bmiller/projects/hapi-login-example-postgres/test/_create_table.js:5:10)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Function.Module.runMain (module.js:447:10)

npm ERR! Linux 3.10.0-327.10.1.el7.x86_64
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "run" "create"
npm ERR! node v5.9.0
npm ERR! npm  v3.7.3
npm ERR! code ELIFECYCLE
npm ERR! hapi-login-example-postgres@1.0.14 create: `node test/_create_table.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the hapi-login-example-postgres@1.0.14 create script 'node test/_create_table.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the hapi-login-example-postgres package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node test/_create_table.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs hapi-login-example-postgres
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls hapi-login-example-postgres
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/bmiller/projects/hapi-login-example-postgres/npm-debug.log

```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "hapi-login-example-postgres",
-  "version": "1.0.14",
+  "version": "1.0.15",
   "description": "Example of building a registration/login flow using Hapi.js and PostgreSQL",
   "main": "index.js",
   "directories": {

--- a/test/_create_table.js
+++ b/test/_create_table.js
@@ -1,8 +1,10 @@
 require('env2')('./config.env'); // see: https://github.com/dwyl/env2
 var assert = require('assert');
+var path = require('path');
 // avoid having pg as a dependency by simply requiring the nested dependcy
-var _pg = '../node_modules/hapi-postgres-connection/node_modules/pg/lib/index.js';
-var pg = require(_pg); // manual connection just for this
+var _pg = '/../node_modules/hapi-postgres-connection/node_modules/pg/lib/index.js';
+console.log(path.resolve(__dirname + _pg));
+var pg = require(path.resolve(__dirname + _pg)); // manual connection just for this
 
 function create_tables (callback) {
   var client = new pg.Client(process.env.DATABASE_URL);

